src/test/java/org/jabref/gui/preferences/journals/JournalAbbreviationsViewModelTabTest.java

```diff
@@ -515,13 +511,6 @@ void testSaveAbbreviationsToFilesCreatesNewFilesWithWrittenAbbreviations(TestDat
         assertEquals(testData.finalContentsOfFile3, actual);
     }
 
-    @ParameterizedTest
-    @MethodSource("provideTestFiles")
-    void testSaveExternalFilesListToPreferences(TestData testData) throws IOException {
-        addFourTestFileToViewModelAndPreferences(testData);
-        verify(preferencesService).storeJournalAbbreviationPreferences(any());
-    }
-
```

Comment:
From reading the deleted code only, I don't get why it is not replaced by some other testing code - 
Shouldn't the preferences then stored be in the `abbreviationPreferences`?

Reply:
Thing is, it does not make sense to test here if the private putBoolean method is called. This should be a test in JabRefPreferencesTests or Journal abbreviationPreferencesTest, but this is somewhat trivial and I assume the behavior of observers is testet within javafx already.

If you really want to test the view model we should certainly not test with any() as an argument.

With other words: the test, as it is, is useless imho.